web/frontend/src/components/slam-2d-render.vue

```diff
@@ -78,6 +108,21 @@ const disposeScene = () => {
   scene.clear();
 };
 
+// Find the desired color bucket for a given probability. This assumes the probability will be a value from 0 to 100
+const probToColorMapBucket = (normProb: number, numBuckets: number): number => {
+  const prob = Math.max(Math.min(100, normProb * 255), 0);
+  return Math.floor((numBuckets - 1) * prob / 100);
+};
+
+/*
+ * Map the color of a pixel to a color bucket value.
+ * normProb is the probability value normalized by the size of a byte(255) to be between 0 to 1.
+ */
+const colorBuckets = (normProb: number): THREE.Vector3 => {
```

Comment:
Can this also be unit tested?

Reply:
added a ticket to unit test this https://viam.atlassian.net/browse/RSDK-2606